invenio_rdm_records/services/pids/tasks.py

```diff
@@ -9,8 +9,16 @@
 
 from celery import shared_task
 
+from invenio_rdm_records.proxies import current_rdm_records
+
 
 @shared_task(ignore_result=True)
-def register_pid(recid, pid_type):
-    """Register a PID."""
-    pass
+def register_pid(pid, scheme, record_id, provider_name=None):
```

Comment:
ocd: would style call it pid_type instead of scheme since a record level is what we use... scheme is more at config level because is more understandable for the users.

Also maybe change the order of record_id and scheme to have the related ones closer?